cql/src/main/resources/cql.yaml

```diff
@@ -0,0 +1,126 @@
+
+# CQL transport config YAML
```

Comment:
It looks like this file is not referenced anywhere. Is it intended as an example?

Reply:
Yes, it's an an example file.